### PR TITLE
C++: More IR QLDoc (including `Opcode.qll`)

### PR DESCRIPTION
**Pull request by dbartol**
This PR adds QLDoc for every opcode in `Opcode.qll`. It's is all copied from the corresponding `Instruction.qll` QLDoc via a Python script, included as part of the PR.

There are a few preparatory commits to clean up some existing QLDoc comments, add a few missing `Instruction` classes, etc.

### CODE REVIEW

**Comment by dbartol** [comment]
I've implemented the changes we discussed in today's C++ analysis team meeting, so now the `Opcode` docs just refer to the corresponding `Instruction` docs.

**Comment by dbartol** [comment]
Since this PR hadn't been reviewed yet anyway, I've added QLDoc for several more files.

File: cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll [hunk 1]
```diff
@@ -1880,6 +1988,10 @@ class BuiltInOperationInstruction extends Instruction {
     operation = Raw::getInstructionBuiltInOperation(this)
   }
 
+  /**
+   * Gets the language-specific `BuildInOperation` object that specifies the operation that is
```
Comment (jbj) [inline cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll:1992]: `s/BuildInOperation/BuiltInOperation/`

**Comment by dbartol** [comment]
Thanks. I'll address all of the above feedback in a follow-up PR.

**Pull request merged**